content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1107,6 +1174,13 @@ after the operation: the built-in Job controller and the external controller
 indicated by the field value.
 {{< /warning >}}
 
+{{< note >}}
```

Comment:
Note not needed in this section, as it's not really related to the section's topic

Reply:
I agree, it can be confusing. The only purpose of this was to reference that the logic change is controlled by the feature gate. Now I propose to mention this as a note in one place only.